src/esp/assets/managers/ObjectAttributesManager.cpp

```diff
@@ -142,21 +136,21 @@ int ObjectAttributesManager::registerAttributesTemplateFinalize(
     // If renderAssetHandle is neither valid file name nor existing primitive
     // attributes template hande, fail
     // by here always fail
-    LOG(ERROR) << "ObjectAttributesManager::registerAttributesTemplateFinalize "
-                  ": Render asset template handle : "
-               << renderAssetHandle
-               << " specified in object template with handle : "
-               << objectTemplateHandle
-               << " does not correspond to existing file or primitive render "
-                  "asset.  Aborting. ";
+    LOG(ERROR)
+        << "ObjectAttributesManager::registerAttributesTemplateFinalize "
+           ": Render asset template handle : "
+        << renderAssetHandle << " specified in object template with handle : "
+        << objectTemplateHandle
+        << " does not correspond to any existing file or primitive render "
+           "asset.  Aborting. ";
     return ID_UNDEFINED;
   }
 
   if (assetAttributesMgr_->getTemplateLibHasHandle(collisionAssetHandle) > 0) {
     // If collisionAssetHandle corresponds to valid/existing primitive
     // attributes then setCollisionAssetIsPrimitive to true
     objectTemplate->setCollisionAssetIsPrimitive(true);
-  } else if (Corrade::Utility::Directory::exists(collisionAssetHandle)) {
+  } else if (this->isValidFileName(collisionAssetHandle)) {
```

Comment:
If we override this, let's also provide a warning log instead of doing so silently.